core/src/main/scala/org/bykn/bosatsu/MainModule.scala

```diff
@@ -1018,16 +1018,14 @@ abstract class MainModule[IO[_]](implicit
       def runEval: IO[(Evaluation[Any], Output.EvaluationResult)] = withEC {
         implicit ec =>
           for {
-            pn <- inputs.packMap(this, List(mainPackage), errColor)
-            (packs, names) = pn
-            mainPackageNameValue <- mainPackage.getMain(names)
-            (mainPackageName, value) = mainPackageNameValue
+            (packs, names) <- inputs.packMap(this, List(mainPackage), errColor)
+            (mainPackageName, value) <- mainPackage.getMain(names)
             out <-
               if (packs.toMap.contains(mainPackageName)) {
                 val ev = Evaluation(packs, Predef.jvmExternals)
 
                 val res = value match {
-                  case None        => ev.evaluateLast(mainPackageName)
```

Comment:
we should just remove evaluateLast and rewrite to `evaluateMain`. `Last` is how, `Main` is what.